src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletStorage.scala

```diff
@@ -106,10 +106,10 @@ final class WalletStorage(store: LDBKVStore, settings: ErgoSettings)
     * Read state context from the database
     * @return state context read
     */
-  def readStateContext: ErgoStateContext = store
+  def readStateContext(parameters: Parameters): ErgoStateContext = store
```

Comment:
Likely it will be cleaner to serialize parameters (and we have serializer already in "object ParametersSerializer"), and during deserialization check if parameters there, and fallback to LaunchParameters if so (so old versions of the node storage will be supported).

Reply:
Frankly I don't understand that as we are passing `LaunchParameters` from `ErgoApp`. It is all `LaunchParameters`. Do you mean trying to serialize `parameters` at startup in `ErgoApp` to check if they are valid?